zipline/algorithm.py

```diff
@@ -1364,13 +1364,18 @@ def set_long_only(self):
     ##############
     @api_method
     @require_not_initialized(AttachPipelineAfterInitialize())
-    def attach_pipeline(self, pipeline, name):
+    def attach_pipeline(self, pipeline, name, chunk_size=None):
```

Comment:
`pd.read_csv` has `chunksize` with no underscore.  Idk if it's worth trying to be consistent?